### DISCUSSION TITLE
How to generate the configuration file only avoiding indexing?

### DISCUSSION THREAD

**Discussion by debankanroy** [oracle/opengrok#4033] [Q&A]
Hi @vladak / community,
I would like to generate the configuration file only without the indexing. Is there any known way to achieve this? I want to generate the configuration file and let the web application know.... I have a plan to handle the actual  indexing part separately.

I was following this: 
https://github.com/oracle/opengrok/wiki/How-to-setup-OpenGrok

Step.3 - Indexing
This step consists of these operations:

create index
**let the indexer generate the configuration file
notify the web application that new index is available**

**Comment by vladak** [comment]
Try using the -n and -U indexer options.
